41_advanced_node_management/20_extend_node_inventory.txt

```diff
@@ -0,0 +1,100 @@
+[[extend-nodes-inventory, Extend node inventory]]
+
+=== Extend node inventory
+
+
+It is quite common to need to gather information on your nodes that are not present
+in the standard Rudder inventory information. 
+
+As of Rudder 4.3.0, you can add get more information about a node thanks to 
+`inventory hooks`. These information will be available as `node properties`.
```

Comment:
maybe `*node properties*`?

Reply:
no preference, and we don't really have a convention for that. OK for *